File: Python/Matrix or 2D Array  - print using while loop.py
A = [[1, 2, 3],
     [4, 5, 6],
     [7, 8, 9]]

print("Matrix: A")
row = 0
while (row < len(A)):
    col = 0
    while (col < len(A[row])):
        print("\t", A[row][col], end='')
        col += 1
    row += 1
    print()

File: Java/Matrix or 2D Array  - print using while loop.java
class matrixPrint_whileLoop {
    public static void main(String[] args){
        int[][] A = {
            { 1, 2, 3 },
            { 4, 5, 6 },
            { 7, 8, 9 }
        };
        System.out.println("Matrix: A");
        int row = 0;
        while (row < A.length) {
            int col = 0;
            while (col < A[row].length) {
                System.out.print("\t" + A[row][col]);
                col++;
            }
            row++;
            System.out.println();
        }
    }
}